src/whos-in-charge-now.md

```diff
@@ -0,0 +1,99 @@
+# Who's in charge now?
+
+<style>
+img[alt~="center"] {
+  display: block;
+  margin: 0 auto;
+}
+.columns {
+    display: grid;
+    grid-template-columns: repeat(2, minmax(0, 1fr));
+    gap: 1rem;
+}
+</style>
+
+---
+
+
+## The Rust Foundation
+
+> ... is an independent non-profit organization dedicated to stewarding the Rust programming language, nurturing the Rust ecosystem, and supporting the set of maintainers governing and developing the project.
+
+---
+
+## Foundation Members
+
+![Platinum Members center width:1000](./platinum-members.png)
+
+---
+
+![bg fit](./other-members.png)
+
+---
+
+## The Rust Project
+
+https://www.rust-lang.org/governance
+
+<div class="columns">
+<div>
+
+* Core Team
+* Compiler Team
+* Crates.io Team
+* Dev Tools Team
+* Infrastructure Team
+
+</div>
+<div>
+
+* Language Team
+* Library Team
+* Moderation Team
+* Release Team
+
+</div>
+</div>
+
+---
+
+## Working Groups
+
+<div class="columns">
+<div>
+
+* Async WG
+* Command-line Interface WG
+* Embedded devices WG
+* Game Development WG
+
+</div>
+<div>
+
+* Rust by Example WG
+* Secure Code WG
+* Security Response WG
+* WebAssembly (WASM) WG
+
+</div>
+</div>
+
+---
+
+## Who decides on new features?
+
+* Discuss in chat/forums
+* Open an RFC (a specification)
+* Relevant team takes a vote
+* Tracking ticket is created
+* Pull Request(s) to implement the change
+* Stabilisation
+
+---
```

Comment:
We had a really nice chat about the ecosystem, meetups, and the code of conduct.
I would at least add something about the Code of Conduct, and have some presenter's notes on the meetups/ecosystem if people ask something in that direction.

Reply:
A tricky topic, but I've tried to add some words on this.

Reply:
Straight and to the point. Great!